src/PIL/EpsImagePlugin.py

```diff
@@ -230,6 +230,11 @@ def _open(self) -> None:
         trailer_reached = False
 
         def check_required_header_comments() -> None:
+            """
+            The EPS spec requires that some headers exist.
+            This should be checked after all headers have been read,
+            or at the end of the file if that comes first.
```

Comment:
```suggestion
            This should be checked when the header comments formally end, when image
            data starts or when the file ends, whichever comes first.
```
The previous wording of this is slightly odd to me, because aren't all headers read once you get to the end of the file?